toolium/pageelements/page_element.py

```diff
@@ -81,11 +83,19 @@ def web_element(self):
     def _find_web_element(self):
         """Find WebElement using element locator and save it in _web_element attribute"""
         if not self._web_element or not self.config.getboolean_optional('Driver', 'save_web_element'):
-            # Element will be finded from parent element or from driver
-            base = self.utils.get_web_element(self.parent) if self.parent else self.driver
-            # Find elements and get the correct index or find a single element
-            self._web_element = base.find_elements(*self.locator)[self.order] if self.order else base.find_element(
-                *self.locator)
+            # If the element is encapsulated we use the shadowroot tag in yaml (eg. Shadowroot: root_element_name)
+            if self.shadowroot:
+                # querySelector only support CSS SELECTOR locator
```

Comment:
check that self.locator[0] is a By.CSS_SELECTOR and raise an error if it is another type